addons/debugger/threads.js

```diff
@@ -148,14 +159,13 @@ export default async function createThreadsTab({ debug, addon, console, msg }) {
       const topBlock = thread.target.blocks.getBlock(thread.topBlock);
       const result = [cacheInfo.headerItem];
       if (topBlock) {
-        concatInPlace(result, createBlockInfo(topBlock, null));
+        concatInPlace(result, createBlockInfo(topBlock, -1));
```

Comment:
```suggestion
        concatInPlace(result, createBlockInfo(topBlock, 0));
```

Fixes highlight when top block of script is a breakpoint block